mainwindow.cpp

```diff
@@ -0,0 +1,36 @@
+#include "mainwindow.h"
+#include "./ui_mainwindow.h"
+
+void CenterWindow(QWidget *widget);
+
+MainWindow::MainWindow(QWidget *parent)
+    : QMainWindow(parent)
+    , ui(new Ui::MainWindow)
+{
+    ui->setupUi(this);
+
+    this->setFixedSize(1200,700); //width, height in px
```

Comment:
zrób wyzej stała constexpr auto widthInPixels = 1200 i użyj jej w tej funkcji. 

Dodatkowo this jest tutaj zbedne, poniewaz w klasie mamy zawsze dostep do wskaznika this, i nie trzeba tego jawnie pisac. 
setFixedSize(widthInPixels, ...) powinno zadzialac

Reply:
i nie wiem czy fixedSize w takiej postaci jest dobry. Uzależniłbym to w jakis sposob od rozdzielczosci ekranu. Są chyba funkcje, które pobieraja maxHeight.

Reply:
I jeszcze są konflikty do poprawy

Reply:
Jakie konflikty?

Reply:
![image](https://user-images.githubusercontent.com/36385922/108344691-be56c200-71dd-11eb-81ac-527c304a2cf1.png)